routes/movies.js

```diff
@@ -0,0 +1,44 @@
+const Movie = require('../models/movie')
+const Celebrity = require('../models/celebrity')
+
+const router = require("express").Router(); // What does that mean ?
+
+router.get('/', (req,res,next) => {
+    Movie.find()
```

Comment:
If in the schema you are using Celebrity IDs in the cast array, here you would need to populate to replace the IDs with celebrities.